docs/docs/components.mdx

```diff
@@ -1474,6 +1474,18 @@ Intent classifiers assign one of the intents defined in the domain file to incom
     set `weight_sparsity` to 1 as this would result in all kernel weights being 0, i.e. the model is not able
     to learn.
 
+  * `constrain_similarities`:
+    This parameter when set to `True` applies a sigmoid cross entropy loss over all similarity terms.
+    This helps in keeping similarities between input and negative labels to smaller values.
+    This should help in better generalization of the model to real world test sets.
+
+  * `model_confidence`:
+    This parameter allows the user to configure how confidences are computed during inference. It can take two values:
+    1. `softmax`: Confidences are in the range `[0, 1]` (old behaviour and current default). Computed similarities are normalized with the `softmax` activation function.
+    2. `inner`: Confidences are in the range `[0, 1]`. Computed similarities are normalized with a linear function.
```

Comment:
let's use smth else then `inner`, and resort `inner` for raw inner

Reply:
`normalized_inner`?
I had thought of using `raw_inner`/`unnormalized_inner` for raw inner whenever we add that in.

Reply:
`normalized_inner` also doesn't sound right though, because softmax on top of inner is also some kind of normalization.

Reply:
We could use `relative_inner` for the linear normalized version of `inner` and `absolute_inner` for raw version. What do you think?

Reply:
hm, not sure. `relative_inner` is also not very intuitive

Reply:
@akelad do you have any thoughts on the name?

Reply:
i don't think this name is *that* important tbh...

Reply:
i have 0 opinion on it

Reply:
Changing it to `linear_norm` as discussed on [slack](https://rasa-hq.slack.com/archives/C8LEHMTAM/p1614249074146700?thread_ts=1614160186.107800&cid=C8LEHMTAM).
@akelad I'll ping you once the PR is ready for review again because I'll have to change the docs according to this renaming.